lib/Net/Kubernetes/Role/APIAccess.pm

```diff
@@ -121,30 +130,58 @@ around BUILDARGS => sub {
 
 sub _create_default_base_path {
 	my($self) = @_;
-	return '/api/'.$self->api_version;
+	
+	return '/api/' . $self->api_version;
 }
 
 sub path {
 	my($self) = @_;
 	return $self->url.$self->base_path;
 }
 
+sub _build_server_version {
+	my $self = shift;
+
+	my($version_info) = $self->send_request($self->create_request(GET => $self->url . '/version'));
```

Comment:
Does this get shared across multiple objects from the same client session somehow? Seems like it should.